Lib/test/test_ssl.py

```diff
@@ -3167,7 +3169,7 @@ def test_wrong_cert_tls13(self):
             s.connect((HOST, server.port))
             with self.assertRaisesRegex(
                 ssl.SSLError,
-                'alert unknown ca|EOF occurred|TLSV1_ALERT_UNKNOWN_CA'
+                'alert unknown ca|EOF occurred|TLSV1_ALERT_UNKNOWN_CA|closed by the remote host'
```

Comment:
This can now raise `ConnectionResetError` too (` [WinError 10054] An existing connection was forcibly closed by the remote host` or `[Errno 104] Connection reset by peer`)
```suggestion
                'alert unknown ca|EOF occurred|TLSV1_ALERT_UNKNOWN_CA|closed by the remote host|Connection reset by peer'
```

Reply:
Thank you for your kind correction! I tested the changes you suggested at home and confirmed that they work correctly. 😄